scripts/zkai_attention_kv_stwo_fusion_mechanism_ablation_gate.py

```
@@ -0,0 +1,602 @@
+        if payload.get("section_delta") != expected["section_delta"]:
+            raise FusionMechanismAblationGateError("section delta drift")
+        if payload.get("typed_size_estimate") != expected["typed_size_estimate"]:
+            raise FusionMechanismAblationGateError("typed size estimate drift")
+        if payload.get("controlled_component_grid") != expected["controlled_component_grid"]:
+            raise FusionMechanismAblationGateError("controlled component grid drift")
+        if payload.get("binary_typed_accounting") != expected["binary_typed_accounting"]:
+            raise FusionMechanismAblationGateError("binary typed accounting drift")
+        raise FusionMechanismAblationGateError("payload drift")
+
+    if payload["route_matrix"]["matched_profiles_checked"] != 11:
+        raise FusionMechanismAblationGateError("matched route count drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)